resources/cluster-essentials/files/subscriptions.eventing.kyma-project.io.crd.yaml

```diff
@@ -0,0 +1,186 @@
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: SubscriptionSpec defines the desired state of Subscription
+            properties:
+              filter:
+                description: BebFilters defines the list of BEB filters
+                properties:
+                  dialect:
+                    type: string
+                  filters:
+                    items:
+                      description: BebFilter defines the BEB filter element as a combination
+                        of two CE filter elements
+                      properties:
+                        eventSource:
+                          description: Filter defines the CE filter element
+                          properties:
+                            property:
+                              type: string
+                            type:
+                              type: string
+                            value:
+                              type: string
+                          required:
+                          - property
+                          - value
+                          type: object
+                        eventType:
+                          description: Filter defines the CE filter element
+                          properties:
+                            property:
+                              type: string
+                            type:
+                              type: string
+                            value:
+                              type: string
+                          required:
+                          - property
+                          - value
+                          type: object
+                      required:
+                      - eventSource
+                      - eventType
+                      type: object
+                    type: array
+                required:
+                - filters
+                type: object
+              id:
+                description: ID is the unique identifier of Subscription, read-only.
+                type: string
+              protocol:
+                type: string
+              protocolsettings:
+                description: ProtocolSettings defines the CE protocol setting specification
+                  implementation
```

Comment:
Do you think this is required?

Reply:
It is a valid description of a field. Can you explain your concern in detail?

Reply:
I just think about necessity in having this single words in separate lines.

Reply:
The CRDs are generated [here](https://github.com/kyma-project/kyma/blob/master/components/eventing-controller/config/crd/bases/eventing.kyma-project.io_subscriptions.yaml) by kubebuilder which can be manually changed but I prefer not to.